crates/egui/src/ui.rs

```diff
@@ -1091,8 +1091,9 @@ impl Ui {
     /// # });
     /// ```
     pub fn scroll_with_delta(&self, delta: Vec2) {
-        self.ctx()
-            .input_mut(|input| input.smooth_scroll_delta += delta);
+        self.ctx().frame_state_mut(|state| {
+            state.scroll_delta = delta;
```

Comment:
Are we sure we don't want `+=` here? In either case, we should document that behavior.

Reply:
I changed it to +=, I think that makes more sense. Also documented the behavior.